element.js

```diff
@@ -6,7 +6,11 @@ module.exports = getFormData
 
 function buildElems(rootElem) {
     var hash = {}
-
+    if (rootElem.name) {
+    	hash[rootElem.name] = rootElem
+    	return hash
```

Comment:
If the rootElm has a name attribute cant we assume that none of its children will have a name attribute (unless we're ok with parsing non-compliant markup).